Updates for the first RNAseq lecture (#49)
genome/bfx-workshop: lectures/week_10/README.md

1. Ensure that Docker Desktop is running. 

2. This command will pull the image `rnabio` to your local Docker client with the tag `0.0.1` from the `griffithlab` DockerHub repository:

```bash
docker pull griffithlab/rnabio:0.0.1
```

3. Setup a local workspace directory for the RNAseq course. If you change the path or command used here in Step 3, please update the path to the workspace directory accordingly in Step 4. Also, make a file `test_my_docker_mount` that we will look for later.

```bash
mkdir -p bfx-workshop/rnabio-workspace
echo 'this file helps me test my docker mount' >> bfx-workshop/rnabio-workspace/test_my_docker_mount
```

4. Enter the directory where you created the `rnabio-workspace` folder, and initialize a Docker container using the image we pulled above. `-v` tells Docker to mount our workspace directory within the Docker container as `/workspace` with read-write priveleges. You'll see in the RNAseq course `/workspace` is the base directory for nearly all commands and steps.

```bash
cd bfx-workshop/rnabio-workspace
docker run -v $PWD/:/workspace:rw -it griffithlab/rnabio:0.0.1 /bin/bash
```

> 5. Use `ls` to see what's in this file, enter the `workspace` folder and then use `ls` again to see what is in the `workspace` folder.

Reviewer: ```suggestion
5. Use `ls` to see what's in your current directory, then enter the `workspace` folder and use `ls` again to see what is in the `workspace` folder.
```

1. Switch User `su` to the unbutu user:

```bash
su ubuntu
```

2. Source the pre-installed `.bashrc` file to configure your environment to match the RNAseq course:

```bash
source ~/.bashrc
```

NOTE: Using Docker and the persistent "workspace" volume we attached will allow you to start/stop as you wish. EVERYTIME YOU LOGIN TO THE DOCKER CONTAINER, YOU MUST LOGIN AS THE `ubuntu` USER *AND* `source ~/.bashrc` UPON EACH LOGIN.

### Environment Setup

Create a working directory and set the ‘RNA_HOME’ environment variable
```
mkdir -p ~/workspace/rnaseq/

export RNA_HOME=~/workspace/rnaseq
```

Make sure whatever the working dir is, that it is set and is valid
```
echo $RNA_HOME
```

Since all the environment variables we set up for the RNA-seq workshop start with ‘RNA’ we can easily view them all by combined use of the env and grep commands as shown below. The env command shows all environment variables currently defined and the grep command identifies string matches.
```
env | grep RNA
```

In order to view the contents of this file, you can type:
```
less ~/.bashrc
```

> To exit the file, type `q`.

Reviewer: ```suggestion
To exit `less`, type `q`.
```

### Known Issues/ Discrepancies from RNAbio website

> 1. When running the check strandedness tool in the Module 1, RNAseq Data section, the docker run command cannot be run from within your `griffithlab/rnabio:0.0.1` docker session. To run it, we suggest opening a new tab, `cd` into the `rnaseq` directory you created at the beginning of the course, and using the following command instead-

Reviewer: ```suggestion
1. When running the check strandedness tool in the Module 1, RNAseq Data section, the docker run command cannot be run from within your `griffithlab/rnabio:0.0.1` docker session. To run it, we suggest that you open a new terminal window, `cd` into the `rnaseq` directory you created at the beginning of this assignment, and use the following command instead:
```
